src/bosh_azure_cpi/lib/cloud/azure/models/vm_cloud_props.rb

```diff
@@ -1,20 +1,24 @@
 # frozen_string_literal: true
 
 module Bosh::AzureCloud
+  # https://bosh.io/docs/azure-cpi/#resource-pools
   class VMCloudProps
     include Helpers
-    attr_reader :resource_group_name, :instance_type
-    attr_reader :storage_account_type, :storage_account_kind, :storage_account_max_disk_number, :storage_account_name
+    attr_reader :instance_type, :instance_types
```

Comment:
if it is better to use attr_accessor for the attr which has both  reader and write?

Reply:
For `:instance_type`, will use `attr_accessor`. But for `:availability_zone`, `:availability_set` and `:assign_dynamic_public_ip`, I still prefer `attr_writer` because the writer access is only for test purpose.